src/App.js

```diff
@@ -2,70 +2,122 @@ import {nanoid} from 'nanoid';
 import {useState} from 'react';
 import styled from 'styled-components';
 
+import Footer from "./components/Footer";
+import Header from "./components/Header";
 import InputDataDialog from './components/InputDataDialog.js';
 import { initialInputData } from "./db";
+import { messages } from "./db";
 
 export default function App() {
   
   const [inputs, setInputs] = useState(initialInputData);
+  //const [inputValues, setInputValues] = useState(inputs[0].value);
+  const [showmessages, setShowMessages] = useState(messages[0].text);
   
+  const oldInputLength = initialInputData.length;
+  //alert(initialInputData.length)
   const actualDate = new Date();
-  const formattedActualDate = actualDate.toLocaleDateString()
+  const formattedActualDate = actualDate.toLocaleDateString('en-GB', {year: 'numeric', month: '2-digit', day: '2-digit'})
 
   function updateInput(inputDataValue) {
+    console.log(inputDataValue)
```

Comment:
Please remove unused code and consolelogs before merging into main. The main branch should be as clean as possible.
```suggestion
```